.azure-pipelines-templates/matrix.yml

```diff
@@ -96,6 +105,7 @@ jobs:
         depends_on: ${{ parameters.static_check_job_name }}
 
   # Optimised builds, only run performance tests
+  # When building a release, run all tests and publish install tarball
```

Comment:
does this mean that if the tests fail it will not publish the release?

Reply:
Yes. It is because of the `depends_on` parameter to the `metrics_release.yml` job further down, which waits for the release jobs to have successfully completed.

Reply:
That's an important feature! :)